src/v/storage/probe.cc

```diff
@@ -157,6 +157,11 @@ void probe::setup_metrics(const model::ntp& ntp) {
           [this] { return _bytes_prefix_truncated; },
           sm::description("Number of bytes removed by prefix truncation."),
           labels),
+        sm::make_counter(
+          "compacted_away_bytes",
```

Comment:
Another naming suggestion: maybe `compaction_removed_bytes`?

Reply:
on it